pkg/sql/table.go

```diff
@@ -258,7 +258,7 @@ func (tc *TableCollection) getMutableTableDescriptor(
 	return mutDesc, nil
 }
 
-// resolveSchemaID attempts to lookup the schema from the schemaCache if it exists,
+// ResolveSchemaID attempts to lookup the schema from the schemaCache if it exists,
```

Comment:
the case of the function name in the comment here in 2nd commit does not match the code

Reply:
Done.